test/rebuild-manager.test.js

```diff
@@ -31,7 +31,7 @@ async function setup () {
   return ssb
 }
 
-test('rebuild-manager', t => {
+test.skip('rebuild-manager', t => {
```

Comment:
see thread in rebuild-manager file about removing it

Reply:
drop it!